packages/ui/src/ButtonGroup/index.tsx

```diff
@@ -64,43 +64,37 @@ const isIconOnly = (props: ButtonGroupProps<boolean>): props is ButtonGroupProps
  * the `primary` priority, while subsequent buttons are the `secondary`
  * priority.)
  */
-export const ButtonGroup = ({
-  actionType = 'default',
-  size = 'sparse',
-  ...props
-}: ButtonGroupProps<boolean>) => (
-  <Root $size={size}>
-    {/* Annoying TypeScript workaround — we need to explicitly delineate the
+export const ButtonGroup = ({ actionType = 'default', ...props }: ButtonGroupProps<boolean>) => {
```

Comment:
You might want to review this with "Hide whitespace" enabled — most of the changes in this markup are indentation changes, due to the fact that the component now is wrapped in curly braces due to the `useDensity` hook